### ISSUE TITLE
Add 'Return Focus'

### ISSUE THREAD

**Issue by Hotrian** [Hotrian/OVRdrop-Public#48]
https://steamcommunity.com/app/586210/discussions/0/133261369999249191/#c1694917906658163134

Will detail in the future

**Comment by Crazor** [comment]
Any update on this? I noticed you tagged this "Experimental branch". Did you actually implement something like that?

**Comment by Hotrian** [comment]
Yes this was added in the v1.0.5q10 experimental build. The release notes for that are [over here](https://gist.github.com/Hotrian/7597d97855c155b3971feb64761c9d2c). You can learn how to access the experimental builds on Steam [here](https://github.com/Hotrian/OVRdrop-Public/wiki/Accessing-Alternative-Builds).
